openpype/pipeline/create/context.py

```diff
@@ -1458,6 +1458,65 @@ def create(self, identifier, *args, **kwargs):
                     identifier, label, exc_info, add_traceback
                 )
             ])
+        return result
+
+    def create(
+        self,
+        creator_identifier,
+        variant=None,
+        asset_doc=None,
+        task_name=None,
+        pre_create_data=None
+    ):
+        """Trigger create of plugins with standartized
+
+        Args:
+            creator_identifier (str):
+            variant (str): Variant used for subset name.
+            asset_doc (Dict[str, Any]):
+            task_name (str): Name of task to which is context related.
+            pre_create_data (Dict[str, Any]): Pre-create attribute values.
```

Comment:
Docstring appears to be incomplete? Some descriptions are empty. Plus it should explain what is done/use if None is provided.